### DISCUSSION TITLE
[초보주의] 어떻게 해야할지 모르겠습니다.

### DISCUSSION THREAD

**Discussion by rjc90** [naverz/zepeto-studio-kor#316] [5. Q&A]
안녕하세요. 개발쪽 배워본적도 없는 일반 유저입니다. 
이것저것 따라 해보면서 연습 중 인데...
어려운 것이 있어 질문 드립니다 ㅠ

1. https://assetstore.unity.com/ 에서  Free Rug pack 을 다운받아 제페토 안에 설치 후  캐릭터가 
해당 Rug 앞에 서면, 앉기/눕기 등과 같은 상호작용 표시가 뜨고, 해당 버튼을 누르면 애니메이션이 되도록 하고 싶은데
어떻게 접근을 해야할지 모르겠습니다!

고수님들! 도와주세요!

**Comment by minjujuu** [comment]
안녕하세요! 도움이 될만한 가이드 문서가 있어서 링크 달아봅니다 

Rug 앞에 서면 -> OnTriggerEnter()를 통해 구현할 수 있을 것 같아요 [[ZEPETO Studio - Event functions]](https://studio.zepeto.me/guides/event-functions)

앉기/눕기 등과 같은 상호작용 표시 -> UI 를 먼저 만든 다음 OnTriggerEnter() 이벤트 발생 시 활성화 해주어야 할 것 같아요

해당 버튼을 누르면 -> UI Evnet 중 버튼 onClick 이벤트를 참고하면 좋을 거 같아요 [[ZEPETO Studio - UI Event]](https://studio.zepeto.me/guides/ui-events)

애니메이션 재생 -> Zepeto Character 에서 지원하는 Gesture 함수를 이용해보면 좋을 거 같아요 [[ZEPETO Studio - 2-1. ZepetoCharacter]](https://studio.zepeto.me/guides/zepetocharacter)

**Reply by rjc90** [reply]
감사합니다! 알려주신 링크로 공부해서 만들어 보겠습니다!

**Comment by x4cf** [comment]
상호작용 버튼 -> https://github.com/naverz/zepeto-studio-kor/discussions/157
캐릭터 애니메이션 -> https://github.com/naverz/zepeto-studio-kor/discussions/151
이것도 도움이 될거같아요 :)

**Reply by rjc90** [reply]
감사합니당. 주신 링크도 꼼꼼하게 확인해 보겠습니다.!!!
